src/System.Linq/src/System/Linq/In.cs

```diff
@@ -0,0 +1,38 @@
+﻿//NiceLinq by Mohamed Elshawaf m_shawaf@outlook.com
+using System.Collections.Generic;
+using System.Diagnostics;
+using static System.Linq.Utilities;
+
+namespace System.Linq
+{
+    public static partial class Enumerable
```

Comment:
I'll consider implementing IQuryable instead of IEnumerable, 
I know it's as simple as one can make it directly in code. I just use it and it simplify my code, and I thought it may be a good contribution to LINQ, and, _c'mon_, I'm a junior developer, and it's my first contribution to the open source :)